troubleshooting/shotgun_log_analyzer.rb

```diff
@@ -31,7 +31,7 @@
         query[:page_id] = m[1]
       end
     else
-      m = info.match(/ api_user=(.*?) /)
+      m = info.match(/ api_user=(.*)\s/)
```

Comment:
This only works because api_user is the last param AND there's a space before the --.
the api_user value should really be double quoted if we allow spaces in there but I guess that's a change on the shotgun side.